src/main/scala/com/raphtory/core/deploy/Raphtory.scala

```diff
@@ -23,6 +23,53 @@ import org.apache.pulsar.client.api.Schema
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe._
 
+/**
+  * {s}`Raphtory`
+  *  : `Raphtory` object for creating Raphtory Components
+  *
+  * ## Methods
+  *
+  *   {s}`createGraph(spout: Spout[T] = new IdentitySpout[T](), graphBuilder: GraphBuilder[T], customConfig: Map[String, Any] = Map()): RaphtoryGraph[T]`
+  *    : Create Graph using spout, graph-builder and custom config
+  *
+  *   {s}`createClient(deploymentID: String = "", customConfig: Map[String, Any] = Map())`
+  *    : Create Client
+  *
+  *   {s}`createSpout(spout: Spout[T])`
+  *    : Create Spout
+  *
+  *   {s}`createGraphBuilder(builder: GraphBuilder[T])`
+  *    : Create Graph Builder
+  *
+  *   {s}`createPartitionManager()`
+  *    : Creates Partition Manager
```

Comment:
what is a parition manager exactly? where is it used? what does it do? what type is returned?